default/stringtables/en.txt

```diff
@@ -1213,6 +1213,12 @@ Enable Experimentors
 RULE_ENABLE_EXPERIMENTORS_DESC
 Enables Experimentors during galaxy generation
 
+RULE_EXPERIMENTORS_SPAWN_BASE_TURN
+Experimentors spawn base turn
+
+RULE_EXPERIMENTORS_SPAWN_BASE_TURN_DESC
+Base turn when Experimentors starts to spawn
```

Comment:
should probably explain that a bit more... not clear what it means.

Reply:
I'm not sure either, it base of quite complex formula to calculate actual turn to start spawning.

Reply:
Minimum turn experimentors spawn. Modified by galaxy size and AI Agression?

Reply:
I've used that phase.